composer.json

```diff
@@ -13,6 +13,9 @@
   "require-dev": {
      "codeception/codeception": "2.4.x-dev"
   },
+  "conflicts": {
+    "codeception/stub": "2.0.1"
```

Comment:
This is due to https://github.com/Codeception/Codeception/issues/5092
I'll drop this as soon as a fix is pushed

Reply:
@Slamdunk please drop it :)